kafka-rest/src/main/java/io/confluent/kafkarest/entities/ConsumerInstanceConfig.java

```diff
@@ -69,10 +78,29 @@ public ConsumerInstanceConfig(
         );
       }
     }
+    this.setResponseMinBytes(responseMinBytes);
+    this.setRequestWaitMs(requestWaitMs);
     this.autoOffsetReset = autoOffsetReset;
     this.autoCommitEnable = autoCommitEnable;
   }
 
+  /**
+   * Attaches proxy-specific configurations to the given Properties object
+   */
+  public static Properties attachProxySpecificProperties(Properties props,
+                                                         ConsumerInstanceConfig config) {
+    if (config.getResponseMinBytes() != null) {
+      props.setProperty(KafkaRestConfig.PROXY_FETCH_MIN_BYTES_CONFIG,
+              config.getResponseMinBytes().toString());
+    }
+    if (config.getRequestWaitMs() != null) {
+      props.setProperty(KafkaRestConfig.PROXY_FETCH_MAX_WAIT_MS_CONFIG,
```

Comment:
Ugh, nice catch.

Agreed that we don't want the proxy to use memory when it has the option not to. I think we need to propagate `fetch.max.wait.ms` as well since the configs are tied together in behavior.

e.g suppose we have `fetch.min.bytes=100KB` and `fetch.max.wait=60s` set. The default `fetch.max.wait.ms` is 500 for the AK consumer. The behavior isn't changed - the proxy will receive records every half a second from the consumer and that will use up memory until either `min.bytes` is reached or 60 seconds pass.
Does that make sense?